### PR TITLE
add file

### PR DESCRIPTION
**Pull request by sakettechnocredit**

### CODE REVIEW

File: JavaAssignments/src/saket/swapwiththirdvaiable.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+	int a=10;
+	int b=20;
+	int c;
+	
+	void swap()
+	{
+	c=a;
+	a=b;
+    b=c;
```
Comment (mkanani13) [inline JavaAssignments/src/saket/swapwiththirdvaiable.java]: Alignment issue.

File: JavaAssignments/src/saket/swapwiththirdvaiable.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package saket;
+
+public class swapwiththirdvaiable {
+
+	int a=10;
```
Comment (mkanani13) [inline JavaAssignments/src/saket/swapwiththirdvaiable.java]: Why you need instance variable. as you are using these variables in method only. it should be local variable if usage of variable is limited to method. I suggest use parameters of method .. dont use such hardcode value.

**Review by mkanani13 — CHANGES_REQUESTED** [review]

File: JavaAssignments/src/saket/swapwiththirdvaiable.java (hunk 2 above)
Comment (sakettechnocredit) [reply JavaAssignments/src/saket/swapwiththirdvaiable.java]: done

File: JavaAssignments/src/saket/swapwiththirdvaiable.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+	int swap(int a, int b) {
+		int c;
+		c = a;
+		a = b;
+		b = c;
+		System.out.println("a=" + a);
+		System.out.println("b=" + b);
+
+		return 0;
```
Comment (mkanani13) [inline JavaAssignments/src/saket/swapwiththirdvaiable.java:13]: if you are always returning 0, it doesn't make any sense.
Here you dont need to return anything. you should have return type as void.

**Review by mkanani13 — CHANGES_REQUESTED** [review]

File: JavaAssignments/src/saket/swapwiththirdvaiable.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	};
+
+	public static void main(String[] args) {
+		swapwiththirdvaiable A = new swapwiththirdvaiable();
+		A.swap(5, 10);
```
Comment (mkanani13) [inline JavaAssignments/src/saket/swapwiththirdvaiable.java:18]: instead of 5 and 10 you should use args[0] and args[1].

**Review by mkanani13 — APPROVED** [review]

**Comment by mkanani13** [comment]
Rest code looks good to me hence I merge your request. Still you want to update code please create new branch and update your code based on given comments.

**Pull request merged**